### ISSUE TITLE
setHelpText on MaterialListBox causes problem in 2.1

### ISSUE THREAD

**Issue by mtmmtm9** [GwtMaterialDesign/gwt-material#795]
I upgraded to the 2.1-version. Then i got this exception:

```
ISC_Core.js:1165 *07:51:24.987:WARN:Log:java.lang.AssertionError: This UIObject's element is not set; you may be missing a call to either Composite.initWidget() or UIObject.setElement()
    at Throwable_2(Throwable.java:73)
    at Error_2(Error.java:30)
    at AssertionError_5(AssertionError.java:51)
    at getElement(UIObject.java:560)
    at onAttach_1(Widget.java:348)
    at MaterialWidget_2(MaterialWidget.java:143)
    at getErrorMixin_0(MaterialListValueBox.java:735)
    at setHelperText(AbstractValueWidget.java:94)
    at addEnum(BlockParameterDialog.java:152)
    at addEnumField(BlockParameterDialog.java:132)
    at addField(BlockParameterDialog.java:78)
    at BlockParameterDialog(BlockParameterDialog.java:39)
    at editParameters(BlockContextMenu.java:51)
    at onEvent(BlockContextMenu.java:40)
    at lambda$3_8(ContextMenu.java:119)
    at run_60(ContextMenu.java:119)
    at lambda$4_3(ContextMenu.java:125)
    at execute_20(ContextMenu.java:124)
    at lambda$2_11(ContextMenu.java:110)
    at onClick_18(ContextMenu.java:108)
    at lambda$0_7(MaterialWidget.java:292)
    at onClick_21(MaterialWidget.java:290)
    at dispatch_4(ClickEvent.java:56)
    at dispatch_5(ClickEvent.java:55)
    at dispatch(GwtEvent.java:76)
    at dispatchEvent_0(EventBus.java:40)
    at doFire(SimpleEventBus.java:193)
    at fireEvent_4(SimpleEventBus.java:88)
    at fireEvent_3(HandlerManager.java:127)
    at fireEvent_9(Widget.java:129)
    at fireNativeEvent(DomEvent.java:125)
    at onBrowserEvent(Widget.java:177)
    at dispatchEventImpl(DOM.java:1481)
    at dispatchEvent_2(DOM.java:1420)
    at dispatchEvent_4(DOMImplStandard.java:317)
    at apply_0(Impl.java:309)
    at entry0(Impl.java:361)
    at anonymous(Impl.java:78)
isc.B.push.isc.A.addToMasterLog @ ISC_Core.js:1165
isc_c_Log_addLogMessage @ ISC_Core.js:1164
isc_c_Log_log @ ISC_Core.js:1157
logMessage @ ISC_Core.js:1146
logWarn @ ISC_Core.js:1146
isc_logWarn @ ISC_Core.js:55
logWarn @ SC.java:542
onUncaughtException_0 @ SmartGwtEntryPoint.java:465
reportUncaughtException_1 @ Impl.java:256
reportUncaughtException_0 @ Impl.java:238
entry0 @ Impl.java:363
(anonymous) @ Impl.java:78
```

The problem disappears when removing the call to setHelperText() on a MaterialDropDown-object.
I have a popup-menu which creates these objects when the user selects an item on my popup.
My code worked fine using the 2.0.1-version.

I use these versions (with jitpack):
        <dependency>
            <groupId>com.github.GwtMaterialDesign.gwt-material</groupId>
            <artifactId>gwt-material</artifactId>
            <version>release_2.1-2.1-rc1-gaf98d4c-202</version>
            <scope>provided</scope>
        </dependency>
        <dependency>
            <groupId>com.github.GwtMaterialDesign</groupId>
            <artifactId>gwt-material-addins</artifactId>
            <version>gwt-material-addins-2.1</version>
            <scope>provided</scope>
        </dependency>
        <dependency>
            <groupId>com.github.GwtMaterialDesign</groupId>
            <artifactId>gwt-material-themes</artifactId>
            <version>release_2.1-SNAPSHOT</version>
            <scope>provided</scope>
        </dependency>

**Comment by mtmmtm9** [comment]
The problem is solved using 2.1-SNAPSHOT. 
Thanks.

**Comment by kevzlou7979** [comment]
Cool 👍
